### DISCUSSION TITLE
Where is the glossary?

### DISCUSSION THREAD

**Discussion by kyotarozzz** [WeblateOrg/weblate#5360] [Q&A]
The update removed the glossary management interface from the user management screen.
I searched for the setting of 'Terminology' and the setting of 'Forbidden translations', but I can't find them. 
Unlike the description of the document, 'Mark as terminology' is not found in 'Tools'.

**Comment by nijel** [comment]
Glossary is now regular component, for example https://hosted.weblate.org/projects/weblate/glossary/. Existing content was migrated.

See https://docs.weblate.org/en/latest/user/glossary.html

**Reply by kyotarozzz** [reply]
I checked.
I was concerned about the addition of the component, but I didn't think the content of the term was a component.
It's a confusing interface for old users, but I understand its purpose well.

**Reply by nijel** [reply]
Indeed, it is a big change, but it allows us to bring features currently present only in translation to glossaries - they can have comments, screenshots or can be saved in Git.

We might tweak the UI further to make it easier to find (though the glossaries are more visible now than they used to be).

**Reply by kyotarozzz** [reply]
It is wonderful that we can use the functions available in translation.
Thank you for your great coding.
